opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/zuul/TracePreZuulFilter.java

```diff
@@ -0,0 +1,57 @@
+package io.opentracing.contrib.spring.cloud.zuul;
+
+import com.netflix.zuul.ZuulFilter;
+import com.netflix.zuul.context.RequestContext;
+import io.opentracing.Span;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMapInjectAdapter;
+import io.opentracing.tag.Tags;
+
+class TracePreZuulFilter extends ZuulFilter {
+    static final String COMPONENT_NAME = "zuul";
+    static final String CONTEXT_SPAN_KEY = "span";
```

Comment:
as a value use `TracePreZuulFilter.class.getName()` to be able to identify owner of the property